### DISCUSSION TITLE
[FEATURE REQ] Mass Update Filters

### DISCUSSION THREAD

**Discussion by batman2999** [autobrr/autobrr#1745] [Ideas]
First want to thank the developers of this program. It's pretty great!  Thank you for the effort it takes to make it. 

I have one feature request after using it for a few weeks. Is it possible to add a way to mass update filters?   For example, if you switch downloaders, you have to update each individual filter.  When you have hundreds, this can be tedious. 

Thanks for the consideration!

**Comment by luckylittle** [comment]
Duplicate https://github.com/autobrr/autobrr/discussions/1975

**Reply by zze0s** [reply]
Thanks. Closing this even if it's older since the other one has more comments.

**Comment by zze0s** [comment]
Duplicate https://github.com/autobrr/autobrr/discussions/1975
